dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -34,7 +34,7 @@
         #teacher-name-label{style: "display: none;"}
           = f.label :name, t('activerecord.attributes.user.name').html_safe
         #student-name-label{style: "display: none;"}
-          = f.label :name, t('activerecord.attributes.user.name_example').html_safe
+          = f.label :name, simple_format(h(t('activerecord.attributes.user.name_example')))
```

Comment:
could this instead do something like

```haml
= f.label :name do
    = t('activerecord.attributes.user.name')
    %br
    = t('activerecord.attributes.user.name_example')
  end
```

And then have `name_example` just be `"(e.g. Cool Coder or Jane S.)"`

Reply:
Would that give enough context to translators? I agree that in the code that's much cleaner.